x-pack/plugins/security_solution/public/common/components/alerts_viewer/alerts_table.tsx

```diff
@@ -70,22 +70,24 @@ const AlertsTableComponent: React.FC<Props> = ({
   startDate,
   pageFilters = [],
 }) => {
+  const dispatch = useDispatch();
   const alertsFilter = useMemo(() => [...defaultAlertsFilters, ...pageFilters], [pageFilters]);
   const { filterManager } = useKibana().services.data.query;
-  const { initializeTimeline } = useManageTimeline();
 
   useEffect(() => {
-    initializeTimeline({
-      id: timelineId,
-      documentType: i18n.ALERTS_DOCUMENT_TYPE,
-      filterManager,
-      defaultModel: alertsDefaultModel,
-      footerText: i18n.TOTAL_COUNT_OF_ALERTS,
-      title: i18n.ALERTS_TABLE_TITLE,
-      unit: i18n.UNIT,
-    });
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+    dispatch(
+      timelineActions.initializeTGridSettings({
+        id: timelineId,
+        documentType: i18n.ALERTS_DOCUMENT_TYPE,
+        filterManager,
+        defaultColumns: alertsDefaultModel.columns,
+        excludedRowRendererIds: alertsDefaultModel.excludedRowRendererIds,
+        footerText: i18n.TOTAL_COUNT_OF_ALERTS,
+        title: i18n.ALERTS_TABLE_TITLE,
+        unit: (n: number) => i18n.UNIT(n),
```

Comment:
consider adding a `TODO:  avoid passing this through the store`, which happens via `setInitializeTgridSettings` https://github.com/elastic/kibana/pull/100265/files?diff=split&w=1#diff-8722f79b4ed144ce22175c7374dbba141e719ac7974bd9df9e8191f41ee2e77aR165